src/usersAPI/users-service.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+
+// Path:
+const usersDataBasePath = path.join(__dirname, '..', 'data', 'users.json');
+
+// Array for tests only; user.json file could store all the data permanently.
+let users = [];
```

Comment:
Cleanup your code. If you store data in memory (it's preferable for this task) - do it everywhere. Or store in fs. But do it same way everywhere.